src/components/Article.jsx

```diff
@@ -0,0 +1,30 @@
+import Back from "./Back";
+
+export default function Article({
+  title,
+  description,
+  link,
+  src,
+  viewTrending,
+}) {
+  return (
+    <div className="pt-8 pb-8">
+      <Back onClick={viewTrending} />
+      <h1 className="font-medium text-[36px] leading-[1.2] my-5">{title}</h1>
+      <div>
+        <img src={src} className="w-full aspect-[5/2] object-cover" alt="" />
```

Comment:
As you may know, the image is coming from an API, which means that sometimes the image may not load. That's why it's important to fill in the 'alt' attribute.